src/common/sizer.cpp

```diff
@@ -676,15 +677,34 @@ wxSizer::~wxSizer()
 
 wxSizerItem* wxSizer::DoInsert( size_t index, wxSizerItem *item )
 {
-    m_children.Insert( index, item );
+    // Use the scoped pointer to avoid memory leaking when
+    // wxWindowBase::SetContainingSizer() throws.
+    wxScopedPtr<wxSizerItem> scoped_item( item );
```

Comment:
Let's follow the usual naming conventions
```suggestion
    wxScopedPtr<wxSizerItem> scopedItem( item );
```